be/src/olap/field.h

```diff
@@ -400,6 +400,11 @@ class BitmapAggField: public Field {
         _agg_info->init(dst, (const char*)src.cell_ptr(), false, arena);
     }
 
+    char* allocate_memory(Slice* slice, char* variable_ptr) const override {
```

Comment:
we can write this like this
char* allocate_memory(char* cell_ptr, char* variable_ptr) const override {
auto slice = (Slice*)cell_ptr;
 slice->data = nullptr;
        return variable_ptr;
}